pom.xml

```diff
@@ -691,6 +694,9 @@
             <configuration>
               <failOnWarning>true</failOnWarning>
               <ignoreNonCompile>true</ignoreNonCompile>
+              <ignoredDependencies>
+                <ignoredDependency>com.github.luben:zstd-jni</ignoredDependency>
```

Comment:
Why is this needed?

Reply:
We declare zstd-jni as a dependency even though it's not directly used in our code. If we didn't ignore zstd-jni here analyze-only would fail due to unused dependency.

Reply:
In that case, we can use the `runtime` maven scope?